scripts/setup-certs.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Copied from https://stackoverflow.com/questions/7580508/getting-chrome-to-accept-self-signed-localhost-certificate
+
+######################
+# Become a Certificate Authority
+######################
+
+echo '===== Generate CA private key '
+openssl genrsa -des3 -out certs/myCA.key 2048
```

Comment:
If you remove the `-des3` option, openssl won't ask for a passphrase. 

From the man page (`man openssl genrsa`):
> These options encrypt the private key with specified cipher before outputting it. If none of these options is            specified no encryption is used. If encryption is used a pass phrase is prompted for if it is not supplied via the            -passout argument.